src/compiler/marks.ts

```diff
@@ -31,7 +31,7 @@ export function compileMarks(model: Model): any[] {
       type: MARKTYPES_MAP[mark],
       from: {
         // from.data might be added later for non-facet, single group line/area
-        transform: [{type: 'sort', by: sortBy}]
+        transform: [{ type: 'sort', by: sortBy }]
```

Comment:
I did this automatically. I just right clicked and said "beautify content". It's not enforced, I believe and I definitely don't want this enforced.